### PR TITLE
Remove the x64 configuration in vbc.csproj

### PR DESCRIPTION
**Pull request by jasonmalinowski**
This is old detritus, and didn't actually build. But if you accidentally built from a x64 build window, it'd break you.

Note: this is the desktop command line compiler project. The CoreCLR project is unaffected.

*Reviewers:* @dotnet/roslyn-compiler, @dotnet/roslyn-infrastructure

### CODE REVIEW

**Comment by jasonmalinowski** [comment]
@gafter, this is what you got bit by.

**Comment by davkean** [comment]
:+1:

File: src/Compilers/VisualBasic/vbc/vbc.csproj [hunk 1]
```diff
@@ -34,25 +34,6 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <PlatformTarget>AnyCPU</PlatformTarget>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
-    <DebugSymbols>true</DebugSymbols>
-    <OutDir>..\..\..\..\Binaries\Debug\amd64</OutDir>
-    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
-    <NoWarn>1591</NoWarn>
-    <DebugType>full</DebugType>
-    <PlatformTarget>x64</PlatformTarget>
```
Comment (jaredpar) [inline src/Compilers/VisualBasic/vbc/vbc.csproj]: Is there any reason we should allow leaf projects to customize PlatformTarget?

**Comment by gafter** [comment]
:+1: Thank You!

File: src/Compilers/VisualBasic/vbc/vbc.csproj (hunk 1 above)
Comment (jasonmalinowski) [inline src/Compilers/VisualBasic/vbc/vbc.csproj]: Well, implicitly in this pattern is a proper specification of which project targets this project actually targets. i.e. there is no Platform that you can set (after my change) that will result in PlatformTarget=x64. We could centralize but then we might have other things happening we don't like, I'm not sure.

**Pull request merged**